src/apps/hello-world/hello-world.entry.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import HelloWorld from "./hello-world";
+
+export interface HelloWorldEntryProps {
+  titleText: string;
+  introductionText: string;
+}
```

Comment:
This interface looks a lot like `HelloWorldProps`. Can/should we reuse that type instead?

Reply:
The thing is that an entry is different is different than the actual component (the way it looks like it is architected)
The entry's responsibility is to expose props that can be set as data-* attributes and translated into what the component needs. The hello-world example is funny that way, because the props are so similar. The only difference is that the entry props are suffixed with "Text", which results in a data attribute eg. like: data-title-text.